Kudu.Services.Web/DebugConsole/Default.cshtml

```diff
@@ -1,186 +1,186 @@
-﻿@{

```

Comment:
This always seem to happen when i make changes to this file and the next commit is @badescuga 's. I'll sort this out tomorrow with him. Admittedly i use Vim to edit stuff, so i might be a hipster, the jury is still out on that.